### ISSUE TITLE
[Bug] Pièce jointe n’apparaissent pas dans Actualité 

### ISSUE THREAD

**Issue by ftplv** [PapillonApp/papillon-v7#511]
### Description du bug

Les pièces jointes ne sont pas disponibles dans l’onglet actualité (Skolengo).

### Étapes à reproduire

1. Ouvrir un message sensé comporter une pièce jointe dans l’onglet actualité.
2. Ne pas voir de pièce jointe 

### Comportement attendu

Un bouton permettant de consulter ou de télécharger la pièce jointe

### Appareil

iPhone 6S+, IPad Pro M1

### Version du système d`exploitation

iOS 15.8.3, IPadOS 18.2

### Version

7.6.0

### Environnement

Application native iOS

### Source de l'application

Stores (Play Store / App Store)

### Service scolaire

🟡 Skolengo

### Captures d'écran / vidéo

![image](https://github.com/user-attachments/assets/ae020dfe-5547-45cb-b115-c16d3ee3e35c)

https://github.com/user-attachments/assets/a7876ab0-70da-4a4e-86f2-41d3f7b566ea

**Comment by NathanBnm** [comment]
@ftplv est-ce que tu rencontres toujours le problème ? Je viens de tester avec un compte Skolengo sur Android avec la version 7.7.2 et j'ai bien les pièces jointes.

Par contre les liens ne s'affichent pas mais c'est un autre problème.

**Comment by ftplv** [comment]
> @ftplv est-ce que tu rencontres toujours le problème ? Je viens de tester avec un compte Skolengo sur Android avec la version 7.7.2 et j'ai bien les pièces jointes.
> 
> Par contre les liens ne s'affichent pas mais c'est un autre problème.

Difficile de savoir, l'application n'ouvre plus mes comptes Skolengo et Pronote sur mon iPhone 6S+ depuis la version 7.7.0 (testée sur la .1 aussi et cela ne fonctionne pas non plus, je n'ai pas la .2 encore)

**Comment by ftplv** [comment]
> > @ftplv est-ce que tu rencontres toujours le problème ? Je viens de tester avec un compte Skolengo sur Android avec la version 7.7.2 et j'ai bien les pièces jointes.
> > 
> > Par contre les liens ne s'affichent pas mais c'est un autre problème.
> 
> Difficile de savoir, l'application n'ouvre plus mes comptes Skolengo et Pronote sur mon iPhone 6S+ depuis la version 7.7.0 (testée sur la .1 aussi et cela ne fonctionne pas non plus, je n'ai pas la .2 encore)
> 

Après avoir essayé sur un autres appareil où papillon veut bien s'ouvrir, c'est pire qu'avant car les actualités ne s'affichent plus du tout ! ![image](https://github.com/user-attachments/assets/73edd20c-5e11-443d-9698-99f8687c8588)

**Comment by Kgeek33** [comment]
avec la version 7.10.5 (ou 7.10.4), rencontres tu encore ce problème ?

**Comment by ftplv** [comment]
> avec la version 7.10.5 (ou 7.10.4), rencontres tu encore ce problème ?

Avec iOS 15 et papillon 7.10.4 (idem sous iPadOS 18) les actualités n'ont pas été actualisées depuis 2 mois (cf capture d'écran, la roue de chargement n'est pas figé à cause du screen, elle l'est aussi sur mon écran)![image](https://github.com/user-attachments/assets/b409fa66-a7ed-475d-a3a2-9d9237872363)

Sur ces actualités s'il y a 2 mois le PDF du menu n'apparaît toujours pas comme sur l'Issue d'origine. Sur certaines actualités la pièce jointe apparaît mais quand on clique dessus cela entraine une erreur d'autorisation et d'autres actualités ne présente qu'un immense lien hypertexte au lieu de leur texte et bouton de renvoi

**Comment by Kgeek33** [comment]
Pourrais-tu envoyer une capture d'écran des logs, notamment si tu trouves une erreur ?
Dans : Paramètres (Papillon, en cliquant sur ton profil de la page d'accueil) => À propos de Papillon => Version des dépendances (en bas de la page)

**Comment by ftplv** [comment]
> Pourrais-tu envoyer une capture d'écran des logs, notamment si tu trouves une erreur ?
> Dans : Paramètres (Papillon, en cliquant sur ton profil de la page d'accueil) => À propos de Papillon => Version des dépendances (en bas de la page) 

Bien sûr, je suppose que c'est parmi c'est erreur, si jamais vous avez besoin de plus d'infos, n'hésitez pas à revenir vers moi
![image](https://github.com/user-attachments/assets/b14947d5-a847-4eb4-b9f5-113e131425ed)

**Comment by Kgeek33** [comment]
> > Pourrais-tu envoyer une capture d'écran des logs, notamment si tu trouves une erreur ?
> > Dans : Paramètres (Papillon, en cliquant sur ton profil de la page d'accueil) => À propos de Papillon => Version des dépendances (en bas de la page) 
> 
> Bien sûr, je suppose que c'est parmi c'est erreur, si jamais vous avez besoin de plus d'infos, n'hésitez pas à revenir vers moi
> ![image](https://github.com/user-attachments/assets/b14947d5-a847-4eb4-b9f5-113e131425ed)

Cette erreur de token peut arriver. La seule solution la plus efficace est de se déconnecter et de se reconnecter, tout remarchera normalement :)

**Comment by NathanBnm** [comment]
> Cette erreur de token peut arriver. La seule solution la plus efficace est de se déconnecter et de se reconnecter, tout remarchera normalement :)

Peut-être qu'il faudrait gérer l'erreur en affichant une modale "Session expirée" en demandant à l'utilisateur de se reconnecter ?

**Comment by Kgeek33** [comment]
> > Cette erreur de token peut arriver. La seule solution la plus efficace est de se déconnecter et de se reconnecter, tout remarchera normalement :)
> 
> Peut-être qu'il faudrait gérer l'erreur en affichant une modale "Session expirée" en demandant à l'utilisateur de se reconnecter ?

Ouais mais l'issue qui en parle spécifiquement, c'est #585 :)
